packages/licensing/src/class-manager.php

```diff
@@ -0,0 +1,195 @@
+<?php
+/**
+ * A Terms of Service class for Jetpack.
+ *
+ * @package automattic/jetpack-licensing
+ */
+
+namespace Automattic\Jetpack\Licensing;
+
+use Automattic\Jetpack\Connection\Manager as Connection_Manager;
+use Jetpack_IXR_ClientMulticall;
+use Jetpack_Options;
+use WP_Error;
+
+/**
+ * Class Manager.
+ *
+ * @since ??
+ *
+ * Helper class that is responsible for validating licenses to the current site.
+ */
+class Manager {
+	/**
+	 * Name of the WordPress option that holds all known Jetpack licenses.
+	 *
+	 * @const string
+	 */
+	const LICENSES_OPTION_NAME = 'jetpack_licenses';
```

Comment:
What do you think about storing those inside `jetpack_options`, with the rest of Jetpack options?

Reply:
I think it's preferable to keep it a simple option as the purpose is for partners to set that value rather than us (the value may even be set by partners before the Jetpack plugin is enabled - we don't necessarily know how partners will set it up). Regardless, please let me know if there are any special reasons why we may want to go with `jetpack_options` that I may not be aware of.

Reply:
That makes sense. In general, we prefer to keep everything in `jetpack_options` so it's easier to clean up after ourselves when the plugin is deleted, but in this case that makes sense to keep it that way. 👍